we never want libunwind for runtime backtrace on SIG*, so explicitly disable it
--
diff --git a/cmake/include_libraries.cmake b/cmake/include_libraries.cmake
index 1a0b197..090af47 100644
--- a/cmake/include_libraries.cmake
+++ b/cmake/include_libraries.cmake
@@ -26,6 +26,5 @@ if (ENABLE_SAFEC)
 endif (ENABLE_SAFEC)
 find_package(ICONV QUIET)
 find_package(UUID QUIET)
-find_package(Libunwind)
 find_package(NUMA QUIET)
 find_package(ML QUIET)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4cf2456..5153f35 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -43,7 +43,7 @@ if ( HAVE_ICONV )
     LIST(APPEND EXTERNAL_INCLUDES ${ICONV_INCLUDE_DIR})
 endif ()
 
-if ( HAVE_LIBUNWIND )
+if ( false )
     LIST(APPEND EXTERNAL_LIBRARIES ${LIBUNWIND_LIBRARIES})
     LIST(APPEND EXTERNAL_INCLUDES ${LIBUNWIND_INCLUDE_DIRS})
 endif ()
